core/src/main/java/io/confluent/rest/auth/AuthUtil.java

```diff
@@ -163,21 +163,38 @@ private static ConstraintMapping createConstraint(
     return mapping;
   }
 
+  private static void configureAuthentication(Constraint.Builder constraint,
+                                              RestConfig restConfig) {
+    final List<String> roles = restConfig.getList(RestConfig.AUTHENTICATION_ROLES_CONFIG);
```

Comment:
@apurushotama has this been tested that security functionality works?

Reply:
If I understand right, the CP code doesn't go via this path, the confluent-security-plugins overrides the security code https://confluent.slack.com/archives/C082S55LMKL/p1742407345260459?thread_ts=1742249229.647539&cid=C082S55LMKL